tests/test_gw_repeater.sh

```diff
@@ -55,6 +55,13 @@ check_wsl() {
     RP_EXTRA_OPT=(--expose "${RP_UCC_PORT}" --publish "127.0.0.1::${RP_UCC_PORT}")
 }
 
+get_repater_bridge_mac() {
+    if [ "$#" -eq 1 ]; then
+        REPEATER_BRIDGE_MAC=$(run docker container exec "$1" ifconfig br-lan | grep ether | tr -s ' ' | cut --delimiter=' ' -f 3)
```

Comment:
```suggestion
        REPEATER_BRIDGE_MAC=$(run docker container exec "$1" ifconfig @BEEROCKS_BRIDGE_IFACE@ | grep ether | tr -s ' ' | cut --delimiter=' ' -f 3)
```
`br-lan` is platform specific

Reply:
This file is not a ".in" file so it wouldn't work....

Reply:
This won't work on RDKB then since RDKB uses `brlan0`

Reply:
This script does not run on rdkb, only on Linux with a docker environment.